### PR TITLE
depsify init

### PR DESCRIPTION
**Pull request by mpenet**

### CODE REVIEW

File: .github/workflows/clojure.yml [hunk 1]
```diff
@@ -6,7 +6,32 @@ jobs:
...
+            /root/.m2
+          key: ${{ runner.os }}-${{ hashFiles('**/deps.edn') }}
+
+      - name: Lint
+        run: clojure -T:project lint
+
+      - name: Format
+        run: clojure -T:project format-check
+
+      - name: Test
```
Comment (pyr) [inline .github/workflows/clojure.yml]: Why the two test runs?

File: .github/workflows/clojure.yml [hunk 2]
```diff
@@ -6,7 +6,32 @@ jobs:
...
+        id: test
+        run: clojure -T:project test :kaocha.filter/focus '[:unit]'
```
Comment (pyr) [inline .github/workflows/clojure.yml]: There's no integration vs. unit need here

File: tests.edn [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#kaocha/v1
+ {:reporter [kaocha.report/documentation]
```
Comment (pyr) [inline tests.edn:2]: For a small project like this one you can go directly to the empty map I think

**Review by pyr — APPROVED** [review]
A few comments, great!

File: tests.edn (hunk 3 above)
Comment (mpenet) [reply tests.edn:2]: I simplified, but the doc output is quite nice, especially when stuff breaks

**Review by alexanderjamesking — APPROVED** [review]

**Review by remyrd — APPROVED** [review]

**Pull request merged**